Airports/EGPK/Fixes.txt

```diff
@@ -9,6 +9,7 @@ PK12F N055.34.01.160 W004.45.42.480
 PK12I N055.36.08.210 W004.51.53.400
 PK21F N055.35.05.650 W004.30.44.760
 PK30F N055.25.24.550 W004.20.50.300
+PKE01 N055.21.48.570 W004.10.05.640
 PKE02 N055.27.25.590 W004.26.37.650
 PKE02 N055.27.25.590 W004.26.37.650
```

Comment:
```suggestion

```

Reply:
@AleksMax duplicate definition here, if you could remove please :)

Reply:
@hsugden Done :)